experimental/PropertyDDS/packages/property-properties/.eslintrc.cjs

```diff
@@ -13,8 +13,6 @@ module.exports = {
 	},
 	rules: {
 		// Many rules are disabled in PropertyDDS projects. See https://github.com/microsoft/FluidFramework/pull/10272
-		"@typescript-eslint/ban-types": "off",
-		"@typescript-eslint/comma-spacing": "off",
```

Comment:
Note for reviewers: `comma-spacing`, `object-curly-spacing`, and `quotes` were all removed from typescript-eslint as formatting-related rules. With PropertyDDS not being actively maintained, I think it's fine to just lose the rules without replacement.